index.bs

```diff
@@ -2581,6 +2581,38 @@ value explicitly. <em>block-size</em> is defined to be 128
 sample-frames which corresponds to roughly 3ms at a sample-rate of
 44.1 kHz.
 
+<h4 id="AudioNode-tail">AudioNode tail-time</h4>
+
+An {{AudioNode}} can have a <dfn>tail-time</dfn> that is greater than zero. This
+means that even when the {{AudioNode}} is fed silence, non-silent audio can be
+output.
+
+<h4 id="AudioNode-actively-processing">AudioNode lifetime</h4>
+
+{{AudioNode}} can be <dfn id="actively-processing">actively processing</dfn>
+during a <a>render quantum</a>.
+
+- An {{AudioScheduledSourceNode}} is [=actively processing=] when and only when it
+	is playing for at least part of the current rendering quantum.
+- A {{DelayNode}} in a cycle is actively processing only when the absolute value
+	of any output sample for the current [=render quantum=] is greater than or equal
+	to \( 2^{-126} \).
```

Comment:
Why this value?  That's like -700 dB.

Reply:
2^126 is FLT_MIN, the smallest magnitude representable in IEEE 754 binary32 format without loss of precision.

Web audio uses 32-bit float sample format for "High dynamic range", presumably to provide support for values beyond what would be audible (which would be representable with merely 16-bit formats).  We don't know what gain might be applied to the signal, nor to what parameter the signal may be connected.

Implementations already often flush values smaller than FLT_MIN to zero in order to avoid potential performance problems with some hardware.  This is not currently specified, but FLT_MIN is chosen to define a cutoff that is likely already used for other reasons, rather than adding any new arbitrary cutoff.

I suspect FLT_TRUE_MIN would be harder for implementations to support with acceptable performance on all common hardware.  There is also significant loss of precision near this value, which can mean that the value is actually never reached in decay loops.